intranet/templates/itemreg/home.html

```diff
@@ -108,7 +108,7 @@ <h2>
       <li class="phone">
         <b>Phone:</b> {{ phone.manufacturer_name }} {{ phone.model }}
         <br>
-        Serial: {{ phone.serial }}
+        IMEI: {{ phone.imei }}
```

Comment:
You may also want to add information on how to find the IMEI number, both here and on the search page, and link to popular manufacturers' pages on how to do so.

* Apple: https://support.apple.com/en-us/HT204073
* Samsung: https://www.samsung.com/us/support/answer/ANS00048604/
* Google: https://support.google.com/store/answer/3333000?hl=en

Generically, look on the back of the phone, or enter `*#06#` on the dialpad